pkg/cloud/packet/client.go

```diff
@@ -132,3 +137,46 @@ func (p *PacketClient) GetDeviceByTags(project string, tags []string) (*packngo.
 	}
 	return nil, nil
 }
+
+// CreateIP reserves an IP via Packet API. The request fails straight if no IP are available for the specified project.
+// This prevent the cluster to become ready.
+func (p *PacketClient) CreateIP(namespace, clusterName, projectID, facility string) (net.IP, error) {
+	req := packngo.IPReservationRequest{
+		Type:                   packngo.PublicIPv4,
+		Quantity:               1,
+		Facility:               &facility,
+		FailOnApprovalRequired: true,
+		Tags:                   []string{namespace + "/" + clusterName},
+	}
+
+	r, resp, err := p.ProjectIPs.Request(projectID, &req)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode == http.StatusUnprocessableEntity {
+		return nil, fmt.Errorf("Impossible to process the request caused by ip limitation. Please rech out to support.")
```

Comment:
Typo: "reach out"

Also, the message probably should be a little friendlier. Maybe, "Could not create an Elastic IP due to quota limits on the account. Please contact Packet support."